src/PowerShell/PowerShellManager.cs

```diff
@@ -204,42 +205,48 @@ internal void InvokeProfile(string profilePath)
             FunctionInvocationPerformanceStopwatch stopwatch)
         {
             var outputBindings = FunctionMetadata.GetOutputBindingHashtable(_pwsh.Runspace.InstanceId);
-
-            var durableController = new DurableController(functionInfo.DurableFunctionInfo, _pwsh);
+            var durableFunctionsUtils = new DurableController(functionInfo.DurableFunctionInfo, _pwsh, Logger);
 
             try
             {
-                durableController.BeforeFunctionInvocation(inputData);
+                durableFunctionsUtils.InitializeBindings(inputData, out bool hasExternalDFsdk);
+                Logger.Log(isUserOnlyLog: false, LogLevel.Trace, String.Format(PowerShellWorkerStrings.UtilizingExternalDurableSDK, hasExternalDFsdk));
 
                 AddEntryPointInvocationCommand(functionInfo);
                 stopwatch.OnCheckpoint(FunctionInvocationPerformanceStopwatch.Checkpoint.FunctionCodeReady);
 
-                SetInputBindingParameterValues(functionInfo, inputData, durableController, triggerMetadata, traceContext, retryContext);
+                var orchestrationParamName = durableFunctionsUtils.GetOrchestrationParameterName();
+                SetInputBindingParameterValues(functionInfo, inputData, orchestrationParamName, triggerMetadata, traceContext, retryContext);
                 stopwatch.OnCheckpoint(FunctionInvocationPerformanceStopwatch.Checkpoint.InputBindingValuesReady);
 
-                if (!durableController.ShouldSuppressPipelineTraces())
-                {
-                    _pwsh.AddCommand("Microsoft.Azure.Functions.PowerShellWorker\\Trace-PipelineObject");
-                }
-
                 stopwatch.OnCheckpoint(FunctionInvocationPerformanceStopwatch.Checkpoint.InvokingFunctionCode);
                 Logger.Log(isUserOnlyLog: false, LogLevel.Trace, CreateInvocationPerformanceReportMessage(functionInfo.FuncName, stopwatch));
 
                 try
                 {
-                    return durableController.TryInvokeOrchestrationFunction(out var result)
-                                ? result
-                                : InvokeNonOrchestrationFunction(durableController, outputBindings);
+                    if(functionInfo.DurableFunctionInfo.IsOrchestrationFunction)
+                    {
+                        return durableFunctionsUtils.InvokeOrchestrationFunction();
+                    }
+                    else
+                    {
+                        var isActivityFunction = functionInfo.DurableFunctionInfo.IsActivityFunction;
+                        if (!isActivityFunction)
+                        {
+                            _pwsh.AddCommand("Microsoft.Azure.Functions.PowerShellWorker\\Trace-PipelineObject");
```

Comment:
Could you please make this `"Microsoft.Azure.Functions.PowerShellWorker\\Trace-PipelineObject"` a constant?